docs/docs/platform/step-filter.md

```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 9
+---
+
+# Step Filter
+
+Workflow step filtering allows you to customize the flow of notifications in your
+workflow by specifying criteria for which notifications should be delivered. You can use this
+filter to ensure that only relevant notifications are sent to your subscribers,
+improving the efficiency and effectiveness of your communication.
+
+## On Filter
+
+- On payload: You can specify a list of keywords or values that the 'payload' field must
+  match in order for the notification to be delivered. his is useful if you only want to
+  receive notifications that contain certain information.
+
+- On subscriber: You can filter which subscribers receive notifications based on subscriber
+  information. This means that you can send notifications to specific individuals or groups
+  rather than all subscribers..
+
+- On webhook: You can use this feature to provide information about subscribers from a specific
+  webhook and use that information to filter the notification flow. For example, if your webhook
+  returns a field indicating whether a subscriber is currently online, you can use this field
+  to send notifications only to online subscribers.
+
+### Steps to set filter
+
+1. Create a step in your notification workflow. This step will be the location where
+   you will apply your filter.
+
+2. Add a filter to the step. This will allow you to specify the criteria that must be
+   met for the notification to be delivered.
+
+   <div align="center">
+     <a href="https://novu.co" target="_blank">
+     <picture>
+       <source media="(prefers-color-scheme: dark)" srcset="/img/platform/filter/dark-filter-section.png"/>
+       <img src="/img/platform/filter/light-filter-section.png" width="1280" alt="Logo"/>
+     </picture>
+     </a>
+   </div>
+
+3. If you want to add multiple filters, you will need to choose a logical condition
+   of AND or OR (1). This will determine how the filters are applied to the notification.
```

Comment:
```suggestion
   between AND and OR (1). This will determine how the filters are applied to the notification.
```
I guess we only allow those 2 options so far right now.

Reply:
Yea at the moment OR/AND.